odoo_initializer/models/product_loader.py

```diff
@@ -5,6 +5,3 @@ class ProductLoader(BaseLoader):
     model_name = "product.template"
     folder = "product"
     filters = {}
-    field_rules = {
-        "lst_price": "NO_UPDATE"
```

Comment:
The `lst_price` is not part of the `product.template` model right, but rather `product.product`?
If not, we might want to keep the NO_UPDATE rule on `lst_price` and `standard_price` right?

Reply:
What about this comment?

Reply:
Both fields (`lst_price` and `standard_price`) are avilable in both model (`product.template` and `product.product`)

Reply:
Yes, but the rule should stay here right? Otherwise reprocessing the files on top of existing db will override the prices.
Since prices are "data", we just want to import them once.